CONTRIBUTING.md

```diff
@@ -1,303 +1,61 @@
-  - [Discuss big changes as Issues first](#discuss-big-changes-as-issues-first)
-  - [Pull Requests always welcome](#pull-requests-always-welcome)
-  - [Conventions](#conventions)
-    - [Go Code Contributing Guidelines](#go)
-    - [JavaScript  Code Contributing Guidelines](#javascript)
-    - [Git](#git)
-    - [Commit messages](#commit-messages)
-      - [Subject line should not be more than 80 characters long](#subject-line-should-not-be-more-than-80-characters-long)
-      - [A "License" and a "Signed-off-by" trailers are required](#a-license-and-a-signed-off-by-trailers-are-required)
-      - [Commit message examples](#commit-message-examples)
-    - [Code](#code)
-    - [Tests](#tests)
-    - [Documentation](#documentation)
-    - [Pull Requests](#pull-requests)
-    - [Code Review](#code-review)
-    - [Rebasing](#rebasing)
-    - [Merge Approval](#merge-approval)
-- [Credits](#credits)
+Below are instructions on how to contribute to the IPFS project. What does it mean to contribute? Contributing means that you are helping with the coding, design, or writing the documentation of IPFS. 
 
 
-## Security Issues
+### ⏰ Abbreviated Instructions on How to Submit a Pull Request
 
-The IPFS protocol and its implementations are still in heavy development. This means that there may be problems in our protocols, or there may be mistakes in our implementations. And -- though IPFS is not production-ready yet -- many people are already running nodes in their machines. So we take security vulnerabilities very seriously. If you discover a security issue, please bring it to our attention right away!
+* [ ] A little bit shy? Report a bug.
+* [ ] Find out [which issue](https://github.com/ipfs/js-ipfs/issues?q=is%3Aopen+is%3Aissue+label%3Adifficulty%3Aeasy) you want to tackle 
+* [ ] [Fork js-ipfs](https://help.github.com/articles/fork-a-repo/) and start working on [that repo](https://github.com/ipfs/js-ipfs)
+* [ ]  Create a new local branch, make a new pull request
 
-If you find a vulnerability that may affect live deployments -- for example, expose a remote execution exploit -- please send your report privately to security@ipfs.io, please DO NOT file a public issue. The GPG key for security@ipfs.io is [4B9665FB 92636D17 7C7A86D3 50AAE8A9 59B13AF3](https://pgp.mit.edu/pks/lookup?op=get&search=0x50AAE8A959B13AF3).
 
-If the issue is a protocol weakness or something not yet deployed, just discuss it openly.
+#### 📝 Protip:
 
-## Community Guidelines
+Your first issue can be a fixing a spelling mistake or a grammatical error. If want to practice the workflow of contributing try [first-contributions](https://github.com/firstcontributions/first-contributions)
 
-We want to keep the IPFS community awesome, growing and collaborative. We need your help to keep it that way. To help with this we've come up with some general guidelines for the community as a whole:
+_watch our video which will take you step by step in how to submit a patch to IPFS_
```

Comment:
Is there meant to be a link here to a video? Are the italics and lack of capitalization of the sentence intentional?

Reply:
"step by step in how to" => "step by step **through** how to"